recipes/cWB/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -ex
+
+mkdir -pv build
+pushd build
+
+cmake \
+  -DCMAKE_BUILD_TYPE:STRING=Release \
+  -DCMAKE_INSTALL_LIBDIR:PATH="lib" \
+  -DCMAKE_INSTALL_PREFIX:PATH=${PREFIX} \
+  ${SRC_DIR}
```

Comment:
You should use `${CMAKE_ARGS}` which is automatically set by the compiler activation and handles most of the conda-specific arguments you might need:

```suggestion
cmake \
  ${CMAKE_ARGS} \
  -DCMAKE_BUILD_TYPE:STRING=Release \
  ${SRC_DIR}
```